app/assets/javascripts/discourse/app/templates/user/notifications-index.hbs

```diff
@@ -6,15 +6,22 @@
       {{i18n "errors.desc.unknown"}}
     {{/if}}
   </div>
-{{/if}}
-
-<div class="user-notifications-filter-select-kit">
-  {{notifications-filter value=filter onChange=(action (mut filter))}}
-</div>
-
-{{#if hasFilteredNotifications}}
-  {{user-notifications-large notifications=model filter=filter}}
-  {{conditional-loading-spinner condition=loading}}
+{{else if userDoesNotHaveNotifications}}
+  <div class="empty-state">
+    <span class="empty-state-title">{{i18n "user.no_notifications_page_title"}}</span>
+    <div class="empty-state-body">
```

Comment:
why not directly a <p> here ?

Reply:
It's just a first iteration, we're going to style such pages later, I'm using a standard markup that we use in all similar places ([see](https://dev.discourse.org/t/improve-blank-page-syndrome-throughout-discourse/41110)).